crates/cli/src/lib.rs

```diff
@@ -193,6 +199,10 @@ impl Install {
             self.no_default_features,
         )?;
 
+        if !self.bypass_root_check {
+            escalate_root();
```

Comment:
Not what I had in mind.

Was looking for [privilege seperation](https://en.m.wikipedia.org/wiki/Privilege_separation).

When doing `sudo cargo php install` we should fail instantly with a message like 'this should not be executed as root. If you know what you are doing execute it again with the --bypass-root-check flag'.

Only the copy operation should be executed with root privileges.
As this is a simple operation I don't think we need to overcomplicate this and can just use `Command` to do this and guard that for *nix systems.